packages/components/src/card/test/index.js

```diff
@@ -6,12 +6,7 @@ import { mount, shallow } from 'enzyme';
 /**
  * Internal dependencies
  */
-import Card from '../';
-import CardBody from '../body';
-import CardDivider from '../divider';
-import CardFooter from '../footer';
-import CardHeader from '../header';
-import CardMedia from '../media';
+import { Card, CardBody, CardDivider, CardFooter, CardHeader, CardMedia } from '../';
```

Comment:
Seems like those need to go on separate lines - the linter complains 😉

Reply:
Good catch! Somehow the code was not automatically formatted.

Fixed in https://github.com/WordPress/gutenberg/pull/32557/commits/564c683c2a3ff5321d39265075555b995f0746e5